tests/test_visitors/test_ast/test_subscripts/test_forbid_consecutive_slices.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    ForbidConsecutiveSlicesViolation,
+)
+from wemake_python_styleguide.visitors.ast.subscripts import (
+    ConsecutiveSlicesVisitor,
+)
+
+no_consecutive = """
+a = [1, 2, 3, 4]
+b = a[1:]
+"""
+
+no_consecutive_fslice = """
+a = [1, 2, 3, 4]
+b = a[1:][2]
+"""
+
+no_consecutive_sslice = """
+a = [1, [2, 3, 4, 5], 3, 4]
+b = a[1][2:]
+"""
+
+no_slicing = """
+a = [1, 2, 3, 4]
+b = a[1]
+"""
+
+no_slicing_double = """
+a = [1, [5, 6, 7], 3, 4]
+b = a[1][3]
+"""
+
+no_slicing_triple = """
+a = [1, [5, 6, 7, [8, 9, 10]], 3, 4]
+b = a[1][3][1]
+"""
+
+consecutive_double = """
+a = [1, 2, 3, 4]
+b = a[1:][2:]
+"""
+
+consecutive_triple = """
+a = [1, 2, 3, 4, 5, 6]
+b = a[1:][2:][:2]
+"""
+
+consecutive_plus = """
+a = [1, [5, 6, 7, 8, 9, 10], 3, 4]
+b = a[1][2:][:4]
+"""
+
+no_consecutive_for_slices = """
+a = [1, [5, 6, 7, [8, [1, 2, 3, 4], 10]], 3, 4]
+for i in a[1][3][1]:
+    print(i)
+"""
+
+consecutive_for = """
+a = [1, [5, 6, 7, [8, [1, 2, 3, 4], 10]], 3, 4]
+for i in a[1][:4][1:]:
+    print(i)
+"""
+
+
+@pytest.mark.parametrize('code', [
```

Comment:
All these test cases can be parametrized. Here's the basic template:

```python
template = 'a[{0}][{1}][{2}]'
```

Then, we can write cases like:

```python
@pytest.mark.parametrize('code', [
   ('1', '2', '3'), 
   ('1:', '2', '3:'),
   # etc
])
```